exchanges/request/request.go

```diff
@@ -237,3 +265,13 @@ func (r *Requester) SetProxy(p *url.URL) error {
 	}
 	return nil
 }
+
+func (r *Requester) drainBody(body io.ReadCloser) {
+	defer body.Close()
```

Comment:
Is there a particular concern here with the use of `defer`? In Go 1.14 this performance difference is negligible for defer or not: https://golang.org/doc/go1.14#runtime

> This release improves the performance of most uses of defer to incur almost zero overhead compared to calling the deferred function directly. As a result, defer can now be used in performance-critical code without overhead concerns.

Go 1.13 made a lot of headway towards this, with Go 1.14 there is supposed to be no need to avoid `defer`

Reply:
Thanks for the link, I will resolve this!